airflow/executors/base_executor.py

```diff
@@ -85,7 +85,7 @@ def queue_task_instance(
             ignore_task_deps: bool = False,
             ignore_ti_state: bool = False,
             pool: Optional[str] = None,
-            cfg_path: Optional[str] = None):
+            cfg_path: Optional[str] = None) -> None:
```

Comment:
I assume https://issues.apache.org/jira/browse/AIRFLOW-4196 is going to happen soon :)

Reply:
We've already dropped support for Python 2.7 🐍